elasticsearch/pom.xml

```diff
@@ -54,6 +54,31 @@
             <type>test-jar</type>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
> That's a portability issue for the build :(

You're right, I used that as a temporary measure assuming that you were also targeting ES/2-snapshot .. I did assume I could replace that with some Beta tag at least when it would be time to merge your PR, but I'm not finding beta releases :-/ Not sure when they'll release anything

I was also assuming that the switch Lucene 4->5 would be significant in terms of ES compatibility; so it would be nice to avoid some soon-to-be-done rewriting but I guess you're right it's not practical currently to target ES2 yet.. would be good to watch their plans though, if some Beta of ES2 would be coming soon, I'd rather target that ASAP.